students/tim_lurvey/lesson08/circle.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+__author__ = 'Timothy Lurvey'
+
+from math import pi, floor
+
+num = (int, float, complex)
+
+
+class Circle(object):
+    _r = 0.
```

Comment:
this should not be a class variable -- class variable are typically static, not dynamic